.gitlab-ci.yml

```diff
@@ -78,6 +78,19 @@ test-linux-stable:                 &test
     - time cargo test --all --release --verbose --locked
     - sccache -s
 
+check-web-wasm:
+  stage:                           test
+  image:                           tomaka/cargo-web:latest
```

Comment:
I would like to build a docker image for that from our ci - or extend an existing one. @TriplEight what do you think?

Reply:
I'd place this dockerfile inside of our `scripts` repo and include it to dockerfiles CI.